### PR TITLE
Added `url_base` parameter to customize the root url and allow hosting with another application.

### PR DESCRIPTION
**Pull request by nledez**

### CODE REVIEW

**Comment by maddie** [comment]
Not sure about this PR, things like this should be achieved using a reverse proxy

**Comment by nledez** [comment]
Indeed, in most cases, URL rewriting can be done by reverse proxies.

We can also handle static assets by serving them with Nginx if you use it as an RP.
But if you use Haproxy/Traefik, these cannot serve static content.

And last problem, in some RP you can rewrite the content. But this has an impact on the performance.

But as you can see in my last commit [38c153d](https://github.com/librespeed/speedtest-go/pull/41/commits/38c153d4a9113672ffd79400b9f559773378811c), the back end redirects to a static URL.

To summarize, it is a good practice for backends to be able to change the root of their URL.
